src/sprites.py

```diff
@@ -1,20 +1,26 @@
-from src.settings import pygame
-from dataclasses import dataclass
+""" custom sprites classes """
+import pygame
 
 
-@dataclass
```

Comment:
Thanks to the extra pair of eyes, I totally overlooked the Sprite class from the Pygame class itself, what we discussed earlier.